Automattic/wp-calypso
# G Suite: Show badge for admin users on Email page

This pull request updates the [`Email` page](https://wordpress.com/email) to show a badge in front of each G Suite user that has super admin privileges:

![screenshot](https://user-images.githubusercontent.com/594356/99425741-88b2ec00-2903-11eb-803c-e832ed009019.png)

Those changes were extracted from https://github.com/Automattic/wp-calypso/pull/46011 which was getting too big. The badge itself is just a nice-to-have, the main improvement here is that we are now able to determine whether a G Suite user is an admin or not. This information has been returned by the API for several weeks now but had not been surfaced in Calypso until now.

Note this pull request also fixes a warning because of props no longer required, and renames a variable in order to better convey its purpose.

#### Testing instructions

> You can change the role of a G Suite user from the the `Users` section of your [Google Admin](https://admin.google.com/). For a given user, locate and click the `Admin roles and privileges` section, then click the `Roles` section and finally unassign the `Super Admin` role. Please note that we cache information from fifteen minutes to one hour on our side, so you will either have to wait that time to see your changes, or to clear the cache manually (ping me if you need help with this).

1. Run `git checkout add/gsuite-admin-badges` and start your server, or open a [live branch](https://calypso.live/?branch=add/gsuite-admin-badges)
2. Log into a WordPress.com account with a site that has G Suite
3. Open the [`Email` page](http://calypso.localhost:3000/email) for that site
4. Assert that you see a badge in front of each admin user
